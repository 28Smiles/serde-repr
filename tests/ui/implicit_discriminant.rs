use serde_repr::Serialize_repr;

#[derive(Serialize_repr)]
#[repr(u8)]
enum SmallPrime {
    Two,
    Three,
    Five,
    Seven,
}

fn main() {}
