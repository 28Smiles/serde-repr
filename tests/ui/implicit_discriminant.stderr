error: explicit discriminant is required
 --> $DIR/implicit_discriminant.rs:6:5
  |
6 |     Two,
  |     ^^^

error: aborting due to previous error

